src/main/java/edu/rosehulman/covidtracer/service/EmailService.java

```diff
@@ -0,0 +1,120 @@
+package edu.rosehulman.covidtracer.service;
+
+import com.sendgrid.*;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import com.sendgrid.helpers.mail.objects.Personalization;
+import edu.rosehulman.covidtracer.model.Person;
+import edu.rosehulman.covidtracer.repository.PersonRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+
+@Service
+public class EmailService {
+
+    public static final String SERVICE_EMAIL = "rhitcovidtracer@gmail.com";
+
+    @Autowired
+    PersonService personService;
+
+    @Value("${sendgrid.api-key}")
+    private String apiKey;
+
+    @Value("${email.template.basic}")
+    public String basicTemplate;
```

Comment:
These look like new properties, we should add those to the `application.properties.example`, so someone knows they should be filled out if we want emailing to work.

Also, you might want to make that SERVICE_EMAIL configurable too. Seeing as you currently manage that email and if this is ran by the school, they would likely change it to some mailing list at rose.